README.md

```diff
@@ -163,6 +164,93 @@ You simply need to configure Rails to use it in your environment configuration:
 config.action_mailer.delivery_method = :ses # or :sesv2
 ```
 
+## Amazon Simple Email Service (SES) as an ActionMailbox Method
+
+### Configuration
+
+#### Amazon SES/SNS
+
+1. [Configure SES](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/receiving-email-notifications.html) to (save emails to S3)(https://docs.aws.amazon.com/ses/latest/dg/receiving-email-action-s3.html) or to send them as raw messages.
+
+2. [Configure the SNS topic for SES or for the S3 action](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/receiving-email-action-sns.html) to send notifications to +/rails/action_mailbox/amazon/inbound_emails+. For example, if your website is hosted at https://www.example.com then configure _SNS_ to publish the _SES_ notification topic to this _HTTP_ endpoint: https://example.com/rails/action_mailbox/amazon/inbound_emails
+
+#### Rails
+
+1. Configure _ActionMailbox_ to accept emails from Amazon SES:
+
+```
+# config/environments/production.rb
+config.action_mailbox.ingress = :amazon
```

Comment:
sure

Reply:
The only issue is that it'll be a breaking change for people trying to convert to this from the other gem AND I'd have to redo e2e tests

Reply:
I might be able to leave the route as is...tbd

Reply:
(nvm, your previous contradict says to convert to ses)

Reply:
I think it's fair for that breakage since there's one dependency to another. `:ses` is the right config here even if there's a little pain to move over, we should have it right to begin with.